$NetBSD: patch-configure.ac,v 1.2 2015/09/05 23:04:36 yhardy Exp $

Relax version checks so that NetBSD built in libraries can be used.
Builds and runs successfully.

https://github.com/xoreos/xoreos/commit/dd0868b4d4d7de4fbcb3c6d2f4019ccb1d59c899
https://github.com/xoreos/xoreos/commit/d23e00f44f1c03549ff4778b6a9818d07013bfe5

--- configure.ac.orig	2015-07-27 08:05:52.000000000 +0000
+++ configure.ac
@@ -107,8 +107,8 @@ AC_CHECK_FUNCS([strtof])
 
 dnl General purpose libraries
 AX_CHECK_ICONV( , AC_MSG_ERROR([No useable iconv() function found!]))
-AX_CHECK_ZLIB(1, 2, 3, 4, , AC_MSG_ERROR([zlib(>= 1.2.3.4) is required and could not be found!]))
-AX_CHECK_LZMA(5, 0, 5, 2, , AC_MSG_ERROR([liblzma(>= 5.0.5) is required and could not be found!]))
+AX_CHECK_ZLIB(1, 2, 3, 0, , AC_MSG_ERROR([zlib(>= 1.2.3) is required and could not be found!]))
+AX_CHECK_LZMA(5, 0, 3, 2, , AC_MSG_ERROR([liblzma(>= 5.0.3) is required and could not be found!]))
 AX_CHECK_XML2(2, 8, 0, , AC_MSG_ERROR([libxml2(>= 2.8.0) is required and could not be found!]))
 
 dnl Graphic libraries
